wp-api.js

```diff
@@ -224,7 +224,7 @@
 		{
 			idAttribute: 'id',
 
-			urlRoot: WP_API_Settings.root + '/users',
+			urlRoot: WP_API_Settings.root + 'wp/v2/users',
```

Comment:
@adamsilverstein `wp/v2` is the namespace for core endpoints; it could change in newer versions, or with custom code. I wouldn't be against refactoring it out to a property, as long as you can override it in subclasses.